includes/compatibility/ShippingZone.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace MyParcelNL\WooCommerce;
+
+use WC_Shipping_Zone;
+use WC_Data_Store;
+
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+
+class ShippingZone extends WC_Shipping_Zone
+{
+    /**
+     * Constructor for zones.
+     *
+     * @param  int|object|array $zone Zone ID to load from the DB or zone object.
+     */
+    public function __construct( $zone = null ) {
+        if ( is_numeric( $zone ) && ! empty( $zone ) ) {
```

Comment:
Wat doet dit nu precies en wat is er allemaal anders?

Reply:
Je kan vanaf nu een array meegeven zodat de class kan worden opgebouwd zonder db queries uit te voeren.